index.html

```diff
@@ -555,7 +559,7 @@ <h3>Validate Algorithm</h3>
 
       <ol class="algorithm">
         <li>
-Let <strong>credentialToValidate</strong> be a <a>verifiable credentials</a>
+Let <strong>credentialToValidate</strong> be a <a>verifiable credential</a>
```

Comment:
```suggestion
Let <strong>credentialToValidate</strong> be a <a>Verifiable Credential</a>
```

Reply:
The VCWG decided on uncapitalized terminology throughout the vc-data-model specification (example here):

https://www.w3.org/TR/vc-data-model/#ecosystem-overview

We had started with capitalized terms and folks complained that the prose was harder to read, so we switched to lower case terms.